packages/privacy-toolset/src/do-not-sell-dialog/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { FormToggle } from '@wordpress/components';
+
+import './styles.scss';
+
+export type DoNotSellDialogProps = {
+	content: {
+		title: string;
+		longDescription: ( string | React.ReactNode )[];
+		toggleLabel: string;
+		closeButton: string;
+	};
+	onClose: () => void;
+	isActive?: boolean;
+	onToggleActive: ( isActive: boolean ) => void;
+};
+export const DoNotSellDialog = ( {
+	content,
+	onClose,
+	isActive,
+	onToggleActive,
+}: DoNotSellDialogProps ) => (
```

Comment:
This is a personal preference/suggestion, on these types of components I like to try and make it so you have to use the least amount of dot notation in the actual markup itself, just because it looks cleaner. I'd probably put a
```tsx
const { title, longDescription, toggleLabel, closeButton } = content
```
At the top so they can be used as just the variable names. But i reiterate this is just my opinion, feel free to change my mind (or ignore me!) 😄